### PR TITLE
890: Add cardActivation date

### PR DESCRIPTION
**Pull request by f1sh1918**
- activate verification feature flag in build config

resolves https://github.com/digitalfabrik/entitlementcard/issues/890

### CODE REVIEW

**Review by michael-markl — COMMENTED** [review]
I think for stats purposes a `firstActivationDate` would make a bit more sense? (ie. persisting the date when a card was first activated).
Should we change it to `firstActivationDate`?

If not, the graph that I imagine could change in hindsight, which would be weird.

**Comment by f1sh1918** [comment]
> I think for stats purposes a `firstActivationDate` would make a bit more sense? (ie. persisting the date when a card was first activated). Should we change it to `firstActivationDate`?
> 
> If not, the graph that I imagine could change in hindsight, which would be weird.

Okay that makes sense i think. So i adjust the naming and only set it if its null

**Comment by michael-markl** [comment]
Sounds great!

**Review by sarahsporck — APPROVED** [review]
Tested on Android device. Strangely I had to recreate the db for this. I thought exposed creates columns and tables, that do not exist... I hope this does not mess up the Nürnberg card...

**Comment by f1sh1918** [comment]
> 

this could be a good use case for database migrations :)
I also had to recreate the card table. I think there is no real card data in there but we should try to avoid it

**Review by sarahsporck — APPROVED** [review]
This kind of fixes this, although I could not use an old QR-Code for the card activation. Not sure why though...

**Comment by f1sh1918** [comment]
> This kind of fixes this, although I could not use an old QR-Code for the card activation. Not sure why though...

alright no hurry to merge this. I gonna check this next week. Probably it makes more sense to wait until db migrations are ready and write a migration for this instead of using the `schemaUtils`

**Comment by michael-markl** [comment]
I think we should also wait until after the migration PR is merged.

**Comment by f1sh1918** [comment]
> I think we should also wait until after the migration PR is merged.

i updated this with my first migration :)

File: backend/src/main/kotlin/app/ehrenamtskarte/backend/migration/migrations/V0003_AddFirstActivationDate.kt
Comment (michael-markl) [inline backend/src/main/kotlin/app/ehrenamtskarte/backend/migration/migrations/V0003_AddFirstActivationDate.kt:22]: I am not entirely sure, whether semicolon is necessary here.

```suggestion
            where "totpSecret" is not null;
```

**Review by michael-markl — APPROVED** [review]
Can you merge in the latest main into this PR before merging? The CI check is on main but not on this PR yet.

**Pull request merged**
